test_calcuator.py

```diff
@@ -0,0 +1,57 @@
+"""Calculator Tests."""
```

Comment:
You are missing many important test scenarios. It is almost always important not just to test things individually but to make sure they work together in combination. In particular for the calculator a major thing you need to test is that if you have multiple operations they get performed in the right order.